projects/storefrontapp-e2e-cypress/cypress/helpers/product-configuration-overview.ts

```diff
@@ -83,7 +83,7 @@ export function checkNotificationBannerOnOP(
   numberOfIssues?: number
 ): void {
   const resolveIssuesText =
-    ' must be resolved before checkout.  Resolve Issues';
+    ' issues must be resolved before checkout.  Resolve Issues';
```

Comment:
do we really want to check for the exact wording? I see this used with contains. So maybe it's sufficient to check that the the banner contains the text 'Resolve Issues'.

Reply:
This is used to get only the number of issues.
Otherwise the line const issues = text.replace(resolveIssuesText, '').trim(); will give us "8 issues" and the method will fail at the next line. So we had to change that.

This part of the method was never used before and could not have worked prior to our change. I agree with you that we have to remove it from the assertion text, but we have to add " issues" to the replace method otherwise the method will never work